src/js/outreach/views/opt-in_views.js

```diff
@@ -69,25 +74,18 @@ const OptInView = View.extend({
     this.setSubmitButtonState();
   },
   onChangeFirstName() {
-    this.model.set({ first_name: this.ui.firstName.val() });
-    this.setSubmitButtonState();
+    this.model.set({ first_name: trim(this.ui.firstName.val()) });
   },
   onChangeLastName() {
-    this.model.set({ last_name: this.ui.lastName.val() });
-    this.setSubmitButtonState();
+    this.model.set({ last_name: trim(this.ui.lastName.val()) });
   },
   onChangeBirthDate() {
-    this.model.set({ birth_date: this.ui.birthDate.val() });
-    this.setSubmitButtonState();
+    this.model.set({ birth_date: trim(this.ui.birthDate.val()) });
   },
   onChangePhone() {
     const phone = parsePhoneNumber(this.ui.phone.val(), 'US');
```

Comment:
Don't think the phone number needs the `trim()`. Appears that `parsePhoneNumber()` handles all that for us.

`phone` will `undefined` if the user enters only empty spaces. And strings like `"   12345   "` are converted to `"+112345"` automatically.